frontend/src/components/ShootWorkers/WorkerGroup.vue

```diff
@@ -65,81 +67,112 @@ export default {
       return this.machineImagesByCloudProfileName(this.cloudProfileName)
     },
     description () {
+      const { machine, volume, minimum, maximum, maxSurge, zones = [] } = this.workerGroup
+      const machineType = find(this.machineTypes, { name: machine.type })
+      const volumeType = find(this.volumeTypes, { name: get(volume, 'type') })
+      const machineImage = find(this.machineImages, machine.image)
+
       const description = []
-      if (this.workerGroup.machine.type) {
-        const machineType = find(this.machineTypes, { name: this.workerGroup.machine.type })
-        if (machineType) {
-          description.push({
-            title: 'Machine Type',
-            value: machineType.name,
-            description: `(CPU: ${machineType.cpu} | GPU: ${machineType.gpu} | Memory: ${machineType.memory})`
-          })
-          if (machineType.storage) {
-            description.push({
-              title: 'Volume Size',
-              value: `${machineType.storage.size}`
-            })
-          }
-        } else {
-          description.push({
-            title: 'Machine Type',
-            value: this.workerGroup.machine.type
-          })
+      description.push(this.machineTpeDescription(machine, machineType))
+      description.push(this.volumeTpeDescription(machine, machineType, volume, volumeType))
+      description.push(this.volumeSizeDescription(machine, machineType, volume))
+      description.push(this.machineImageDescription(machine, machineImage))
+
+      if (minimum && maximum) {
+        description.push({
+          title: 'Autoscaler',
+          value: `Min. ${minimum} / Max. ${maximum}`
+        })
+      }
+      if (maxSurge) {
+        description.push({
+          title: 'Max. Surge',
+          value: `${maxSurge}`
+        })
+      }
+      if (zones.length) {
+        description.push({
+          title: zones.length > 1 ? 'Zones' : 'Zone',
+          value: join(zones, ', ')
+        })
+      }
+
+      return compact(description)
+    }
+  },
+  methods: {
+    machineTpeDescription (machine, machineType) {
+      if (machineType) {
+        return {
+          title: 'Machine Type',
+          value: machineType.name,
+          description: `(CPU: ${machineType.cpu} | GPU: ${machineType.gpu} | Memory: ${machineType.memory})`
         }
       }
-      if (this.workerGroup.volume) {
-        const volumeType = find(this.volumeTypes, { name: this.workerGroup.volume.type })
+      return {
+        title: 'Machine Type',
+        value: machine.type
+      }
+    },
+    volumeTpeDescription (machine, machineType, volume, volumeType) {
```

Comment:
These helper methods should look something like this:
```js
function getMachineTypeDescription (workerGroup, machineType) {
  const { machine = {} } = workerGroup
  const item = {
    title: 'Machine Type',
    value: machine.type
  }
  if (machineType) {
    const { cpu, gpu, memory } = machineType
    item.description = `(CPU: ${cpu} | GPU: ${gpu} | Memory: ${memory})`
  }
  return item
}
function getVolumeTypeDescription (workerGroup, volumeType, machineType) {
  const { volume = {} } = workerGroup
  if (!volume.type) {
    return
  }
  const item = {
    title: 'Volume Type',
    value: volume.type
  }
  const storageClass = get(volumeType, 'class', get(machineType, 'storage.class'))
  if (storageClass) {
    item.description = `(Class: ${storageClass})`
  }
  return item
}
function getVolumeSizeDescription (workerGroup, machineType) {
  const { volume = {} } = workerGroup
  const size = get(volume, 'size', get(machineType, 'storage.size'))
  if (!size) {
    return
  }
  return {
    title: 'Volume Size',
    value: `${size}`
  }
}
function getMachineImageDescription (workerGroup, machineImage) {
  const { machine: { image = {} } = {} } = workerGroup
  if (!image.name) {
    return
  }
  const item = {
    title: 'Machine Image',
    value: `${image.name} | Version: ${image.version}`
  }
  const expirationDate = get(machineImage, 'expirationDate')
  if (expirationDate) {
    item.description = `(Expires: ${machineImage.expirationDateString})`
  }
  return item
}
```

Reply:
done

Reply:
improved as suggested